README.md

```diff
@@ -45,15 +47,18 @@ capsule-certs-generate --capsule-fqdn "mycapsule.example.com"\
 
 # Copy the ~/mycapsule.example.com-certs.tar to the capsule system
 # register the system to Katello and run:
-capsule-installer --parent-fqdn          "master.example.com"\
-                  --register-in-foreman  "true"\
-                  --foreman-oauth-key    "foreman_oauth_key"\
-                  --foreman-oauth-secret "foreman_oauth_secret"\
-                  --pulp-oauth-secret    "pulp_oauth_secret"\
-                  --certs-tar            "/root/mycapsule.exampe.com-certs.tar"\
+foreman-installer --scenario             "capsule"\
+                  --parent-fqdn           "master.example.com"\
+                  --register-in-foreman   "true"\
+                  --foreman-base-url      "https://master.example.com"\
+                  --trusted-hosts         "master.example.com"\
+                  --trusted-hosts         "mycapsule.example.com"\
+                  --oauth-consumer-key    "foreman_oauth_key"\
+                  --oauth-consumer-secret "foreman_oauth_secret"\
+                  --pulp-oauth-secret     "pulp_oauth_secret"\
+                  --certs-tar             "/root/mycapsule.exampe.com-certs.tar"\
```

Comment:
capsule params will be prefixed when https://github.com/Katello/katello-installer/pull/303 is merged